metadata_backend/api/handlers/submission.py

```diff
@@ -194,7 +206,9 @@ async def get_submission(self, req: Request) -> Response:
 
         LOG.info("GET submission with ID: %r was successful.", submission_id)
         return web.Response(
-            body=ujson.dumps(submission, escape_forward_slashes=False), status=200, content_type="application/json"
+            body=ujson.dumps(submission, escape_forward_slashes=False),
+            status=200,
+            content_type="application/json",
```

Comment:
Interesting that the new code style splits something like this to separate rows...